GameServerConsole/Settings/GameInfo.json.template

```diff
@@ -1,6 +1,7 @@
 {
     "players": [
         {
+            "playerId": 1,
             "rank": "DIAMOND",
             "name": "Test",
             "champion": "Ezreal",
```

Comment:
blowfish key still missing from here. Use the default one `17BLOhi6KZsTtldTsizvHg==` as a default value. (so the debugging will still work easily)

Reply:
Well I should force VS to clear the configs every time i debug..